blueocean-rest-impl/pom.xml

```diff
@@ -130,6 +130,16 @@
             <artifactId>workflow-basic-steps</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
```

Comment:
Are these dependencies still needed after removing node/test related code?

Reply:
Probably not. Removing.